package.json

```diff
@@ -27,9 +27,9 @@
   },
   "homepage": "https://github.com/dashevo/DashJS#readme",
   "dependencies": {
-    "@dashevo/dapi-client": "^0.8.0-dev.14",
-    "@dashevo/dashcore-lib": "^0.18.0",
-    "@dashevo/dpp": "0.10.0-dev.14",
+    "@dashevo/dapi-client": "~0.8.0-dev.14",
+    "@dashevo/dashcore-lib": "~0.18.0",
+    "@dashevo/dpp": "~0.10.0-dev.14",
```

Comment:
Anything newer than 0.10.0-dev.14 (including dev.15) breaks DashJS's ability to register names on Evonet.

Reply:
I think we need dev branch for DashJS as well.